tests/unit/commands/test_remote.py

```diff
@@ -122,6 +153,26 @@ def mock_run_legacy(mocker):
 #############
 
 
+@pytest.mark.parametrize("base", CURRENT_BASES - {"core22"})
```

Comment:
Why are we excluding core22 from this?

Reply:
Because core22 will still use the old remote build, at least for now.